### PR TITLE
us #1762009, #1761008: get build status and abort build that contains execution_id

### PR DESCRIPTION
**Pull request by dmytro-omelian**

### CODE REVIEW

File: plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/factories/ModelCommonFactory.java [hunk 1]
```diff
@@ -61,10 +63,13 @@ public CIJobsList createProjectList() {
 				logger.info("Fetching data for project: " + buildType.getName());
 				if (!ids.contains(buildType.getInternalId())) {
 					ids.add(buildType.getInternalId());
+					List<CIParameter> ciParameters = parametersFactory.obtainFromBuildType(buildType)
+							.stream().filter(parameter -> !parameter.getName().equals(OCTANE_AUTO_ACTION_EXECUTION_ID))
+							.collect(Collectors.toList());
```
Comment (dmytro-omelian) [inline plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/factories/ModelCommonFactory.java]: We have our parameter execution_id that users should not be able to override in Octane, so here we filter parameters that we want to see in Octane
image want to see all except execution_id

Comment (nissimshitrit) [reply plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/factories/ModelCommonFactory.java]: in pipeline and testing framework we are reporting parameters to Octane but we preventing from some system parameters to be shown on UI.
Do you still prefer to filter it on the plugin are you think you can filter it on Octane?
by the way I think you can send parameter to team city job without define it the job itself

File: plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/factories/ModelCommonFactory.java [hunk 2]
```diff
@@ -128,4 +129,9 @@ public CIBuildResult resultFromNativeStatus(Status status, boolean isInterrupted
 		}
 		return result;
 	}
+
+    private String getFullNameFromBuildType(SBuildType buildType) {
+        return buildType.getProjectName() + " - " + buildType.getName();
```
Comment (dmytro-omelian) [inline plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/factories/ModelCommonFactory.java:134]: image

File: plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/factories/ModelCommonFactory.java (hunk 1 above)
Comment (dmytro-omelian) [reply plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/factories/ModelCommonFactory.java]: @nissimshitrit yeah, it is better to filter them on the Octane side, thank you. Removed that part, can you please review it one more time?

File: plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/events/ProgressEventsListener.java [hunk 3]
```diff
@@ -81,6 +83,26 @@ public void buildTypeAddedToQueue(@NotNull SQueuedBuild queuedBuild) {
 		}
 	}
 
+    @Override
+    public void buildRemovedFromQueue(@NotNull SQueuedBuild queuedBuild, User user, String comment) {
```
Comment (dmytro-omelian) [inline plugin/src/main/java/com/hp/octane/plugins/jetbrains/teamcity/events/ProgressEventsListener.java:88]: should check this method cause it works wrong

**Review by nissimshitrit — APPROVED** [review]

**Pull request merged**